docs/demos/connector/neo4j/index.rst

```diff
@@ -1,27 +1,56 @@
 Neo4j
 ====================================
 
-In this folder:
-
-- :doc:`directed-graphsage-on-cora-neo4j-example` and :doc:`undirected-graphsage-on-cora-neo4j-example` provide examples of running GraphSAGE with connection to graph database.
-
-- :doc:`load-cora-into-neo4j` provides demo of loading Cora network citation dataset into neo4j database.
-
-- :doc:`cluster-gcn-on-cora-neo4j-example` provide an example of running Cluster GCN with connection to a graph database.
-
+`StellarGraph <https://github.com/stellargraph/stellargraph>`_ provides various algorithms that can be run on graphs in Neo4J. This folder contains demos of all of them to explain how to connect to Neo4J and how to run these algorithms as part of a TensorFlow Keras data science workflow.
 
 Required Installations:
 
 - **Neo4J**: `Instruction to download <https://neo4j.com/docs/operations-manual/current/installation/>`_.
 - **APOC library** plug-in: APOC provides utilities for common procedures and functions in Neo4J. `Instruction to download and install <https://neo4j.com/developer/neo4j-apoc/>`_.
```

Comment:
not exactly scope of this PR, but with the new clusterGCN I think we need the graphdatascience library also

Reply:
Ah I see, I can quickly add another dot point here with a link to https://neo4j.com/graph-data-science-library